rst_files/testing.rst

```diff
@@ -237,11 +239,12 @@ Dependency Management
 Environments
 ^^^^^^^^^^^^^^^^
 
-As `before <jl_packages>`_, the TOML files define an *environment* for our project, or a set of files which represent the dependency information 
+As `before <jl_packages>`_, the `.toml` files define an *environment* for our project, or a set of files which represent the dependency information 
```

Comment:
Can we define what a TOML is somewhere, because it's not explicitly defined and I didn't really pick up the file extension earlier so I was a bit confused

Reply:
Done.